packages/cli/src/services/auth/login.ts

```diff
@@ -0,0 +1,175 @@
+import { homeDir } from '../../utils'
+import {
+  CoginitoAccess,
+  CoginitoSession,
+  connectSessionVerifyDecodeIdToken,
+  DecodedToken,
+  LoginIdentity,
+  ReapitConnectSession,
+} from '@reapit/connect-session'
+import http from 'http'
+import open from 'open'
+import { writeFileSync, promises, existsSync } from 'fs'
+import 'isomorphic-fetch'
+import decode from 'jwt-decode'
+import ora from 'ora'
+
+export class LoginService {
+  private server
+  private readonly connectOAuthUrl: string = 'https://connect.dev.paas.reapit.cloud' // TODO Change dev to env
+  private readonly port: number = 8080
```

Comment:
this port might not be free, you can use something like https://www.npmjs.com/package/find-free-port to find a spare one